docs/src/pages/docs/crocks/Const.md

```diff
@@ -71,6 +71,29 @@ resetField(changed)
 
 </article>
 
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Const :: c -> Const c a
+```
+
+The constructor for `Const` is a unary function that takes any type as its
+argument. Once constructed, the value will occupy the far left parameter `c`.
+
+```javascript
+import Const from 'crocks/Const'
+
+Const('always and forever')
+//=> Const String a
```

Comment:
I get this structure in the signature, but given the purpose of this new part, what do you think of the idea of this as what would be seen if you ran `inspect` ?

Reply:
Yep, need to address that!
In a couple places. Great 👀